packages/algob/test/fixture-projects/scripts-dir-recursive-cp/scripts/2.js

```diff
@@ -1,7 +1,7 @@
 
 async function run (runtimeEnv, deployer) {
   if (deployer.isDeployMode) {
-    await deployer.deployASC('fee-check.teal', {}, { funder: deployer.accounts[0], microAlgo: 100000000 }, {})
+    await deployer.deployASC('fee-check.teal', {}, { funder: deployer.accounts[0], microAlgo: 100000000, mode: 'Contract Account' }, {})
   }
```

Comment:
is this `mode: 'Contract Account'` or `mode: 'Delegated Approval'` above it is defined as Delegated ?

Reply:
I wanted to keep both modes in some test files. Anyway, reverting to Delegated Approval mode.